### ISSUE TITLE
MultiSelect is incorrectly shown and hidden when used in a SPA view

### ISSUE THREAD

**Issue by jivanova** [telerik/kendo-ui-core#2903]
### Bug report

https://dojo.telerik.com/IzaJIKIr/3

### Reproduction of the problem

1. Click Go to page one link
1. Click Go to page two link

### Current behavior
The select element is not removed. The issue is caused by the **showStart()** method of the view.

### Expected/desired behavior
MultiSelect is removed

Dummy workaround:
viewOne.showStart = function () {
      this.element.parent(".k-multiselect").css("display", "");
}
viewOne.hide = function () {
      this.element.parent(".k-multiselect").detach();
}

### Environment

* **Kendo UI version:** 2023.3.1010
* **Browser:** all

**Comment by hiyelbaz** [comment]
Is there any update on this?

**Comment by nelito987** [comment]
This behavior was originally reported over 8 years ago and has not received significant demand or follow-up from the community since then. Considering this and the availability of a workaround, we are unable to prioritize it over more impactful and widely requested improvements. 

If this limitation is still affecting your scenario, please share additional context or use cases. This will help us better assess the severity and reconsider prioritization in the future. Thank you!
